_posts/2022-02-28-dont-lgtm-code-reviews.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+title: Doesn't look good to me — a requiem for thorough code reviews
+date: 2022-02-28
+tags: code cloudnative consultants deployment continuous-delivery devops patterns review software test testing
+author: Simon Bracegirdle
+image: img/dont-lgtm-code-reviews.jpg
+description:
+  Do you give a cursory review and "LGMT!" on pull requests? Read this to find out how you could bring more rigour to your code review practices and improve the software you deliver.
+---
+
+![Header](/img/dont-lgtm-code-reviews.jpg)
+
+A lot of development teams are doing code reviews these days, it's become the industry norm. But are they done well? The idea of the lazy code review — "LGTM, ship it" — has [become a meme](https://knowyourmeme.com/memes/lgtm), and it's funny because there's an element of truth in it that we've experienced ourselves.
+
+Let's be honest though, when we're working towards deadlines, it's easy to deprioritise or simplify what we see as low priority tasks to ensure we can meet our commitments. This isn't malicious or lazy, it's human nature.
+
+What's the solution then? There's no quick fix of course — it comes down to individuals and teams, and how much value they place in the benefits of code review. But even if you value code reviews highly, you still need a disciplined, well articulated and well understood approach to overcome the habits of individuals and sustain a high level of review quality over time.
+
+In this post i'll take a small step to convince you of the value of thorough code reviews and I'll provide some pointers for bringing rigour and methods to increase the value you get out of them.
```

Comment:
I'll

Reply:
You've buried the lead here.

The first paragraph is your hook - your opportunity to get the reader interested to keep reading.  Would it be better to move this up to the first paragraph?

```suggestion
A lot of development teams are doing code reviews these days, it's become the industry norm. But are they done well? The idea of the lazy code review — "LGTM, ship it" — has [become a meme](https://knowyourmeme.com/memes/lgtm), and it's funny because there's an element of truth in it that we've experienced ourselves..  In this post I aim to convince you of the value of thorough code reviews and I'll provide some pointers for bringing rigour and methods to increase the value you get out of them.

Let's be honest though, when we're working towards deadlines, it's easy to deprioritise or simplify what we see as low priority tasks to ensure we can meet our commitments. This isn't malicious or lazy, it's human nature.

What's the solution then? There's no quick fix of course — it comes down to individuals and teams, and how much value they place in the benefits of code review. 
```

This gives a nice lead into the meat of the post: how you can give me the answers to all my problems!